_glossary/PASSPORTJS.md

```diff
@@ -0,0 +1,7 @@
+# Passport.js
```

Comment:
Do you mind adding a YAML Front Matter like this: https://raw.githubusercontent.com/HugoGiraudel/SJSJ/gh-pages/_glossary/AJAX.md?